tests/shell/tests.py

```diff
@@ -94,3 +122,77 @@ def test_shell_with_bpython_not_installed(self, select):
     # returns EOF and so select always shows that sys.stdin is ready to read.
     # This causes problems because of the call to select.select() toward the
     # end of shell's handle() method.
+
+    def test_get_namespace(self):
+        with self.settings(
```

Comment:
use the `@override_settings` decorator on the test instead, to avoid a level of indentation